src/main/java/devcourse/jpa/domain/item/Item.java

```diff
@@ -0,0 +1,47 @@
+package devcourse.jpa.domain.item;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "item")
+public class Item {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "item_id")
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private int price;
+
+    @Column(nullable = false)
+    private int stockQuantity;
+
+    public Item() {
+    }
```

Comment:
```suggestion
    protected Item() {
    }
```
요런건 protectecd를 주면 불완전한 객체 생성을 막을 수 있어요